pkg/imgutil/imgutil.go

```diff
@@ -398,7 +408,15 @@ func ParseFilters(filters []string) (*Filters, error) {
 		case 1:
 			return nil, fmt.Errorf("invalid filter %q", filter)
 		case 2:
-			if tempFilterToken[0] == FilterBeforeType {
+			if tempFilterToken[0] == FilterDanglingType {
+				if strings.EqualFold(tempFilterToken[1], "true") {
```

Comment:
Not `EqualFold`

```console
$ docker images -f "dangling=TRUE"
Error response from daemon: invalid filter 'dangling=[TRUE]'
```

Reply:
I was intentional about it as it still does not break backward compatibility. LMK if you still want me to change it.

Reply:
Docker seems case-sensitive, so nerdctl should be probably case-sensitive too